app/views/plannings/index.html.erb

```diff
@@ -108,6 +108,7 @@
 
 <%
 controller.js(
+  columns_preferences: current_user.exportable_columns,
```

Comment:
```suggestion
  export_settings: current_user.export_settings,
```

It is no more limited to the columns, so I would prefer to have a more generic naming